docs/content/ingestion/standalone-realtime.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: doc_page
+title: "Realtime Process"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+# Realtime Process
+
+Older versions of Apache Druid (incubating) supported a standalone 'Realtime' process to query and index 'stream pull'
+modes of real-time ingestion. These processes would periodically build segments for the data they had collected over
+some span of time and then set up hand-off to [Historical](../design/historical.html) servers.
+
+This processes could be invoked by
+
+```
+org.apache.druid.cli.Main server realtime
+```
+
+This model of stream pull ingestion was deprecated and removed completely in Druid 0.16.0. Please consider using the
```

Comment:
Could we provide a description here why it was depricated?

Reply:
@egor-ryashin real-time nodes have been deprecated for years. It has been clearly stated for some time now that they will be removed. They suffer from numerous problems and we have not recommended that anyone use them for a long time now.

Reply:
`They suffer from numerous problems`
@fjy I actually would like that we write here what type of problems.

Reply:
I think probably the biggest reason is they are really finicky (every realtime node needs a unique configuration) and also they have some design issues that make it not possible to achieve exactly once ingestion (whereas the kafka/kinesis supervised ingestion methods can do that)

Reply:
Also the windowPeriod thing

Reply:
Thanks for the suggestion @egor-ryashin, this seems reasonable to add to me. I'll try to incorporate some of what @gianm said into the details on this page when I fix conflicts on this branch.